.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-  - 1.5
+  - 1.7
```

Comment:
OK it **should** use the latest patch release but it does not https://travis-ci.org/go-clang/gen/builds/158708830